ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel__downstream-candidate.yaml

```diff
@@ -30,8 +30,8 @@ tests:
       KATA_RPM_BUILD_MD5SUM: 1d108ef4451ce8754eeea08cf12aea0a
       KATA_RPM_BUILD_TASK: "67952818"
       KATA_RPM_VERSION: 3.17.0-1.rhaos4.18.el9_6
-      OPERATOR_INDEX_IMAGE: quay.io/redhat-user-workloads/ose-osc-tenant/osc-fbc-4-18
-      OPERATOR_INDEX_VERSION: on-pr-ca1a164e92ae614eaf8e44e2ca8ade672802adf3
+      OPERATOR_INDEX_IMAGE: quay.io/redhat-user-workloads/ose-osc-tenant/osc-test-fbc
```

Comment:
See [Docs](https://gitlab.cee.redhat.com/osc/documentation/-/merge_requests/58) for how to get OPERATOR_INDEX_IMAGE and OPERATOR_INDEX_VERSION if you want to include it

Reply:
Fixed.